repository.json

```diff
@@ -1,5 +1,5 @@
 {
   "name": "rtl_433 Home Assistant add-ons",
-  "url": "https://github.com/pbkhrv/rtl_433-hass-addons",
-  "maintainer": "Peter Bakhirev"
+  "url": "https://github.com/grinco/rtl_433-hass-addons",
+  "maintainer": "Vadim Grinco"
```

Comment:
I think we need to revert these lines?

Reply:
Yeah, it shouldn't be in the PR - I added it temporarily in my repo after creating the PR to use the latest code with HA - didn't know github wouldn't reference a specific commit.